### ISSUE TITLE
Website keeps loading forever with no errors

### ISSUE THREAD

**Issue by VanRaidex** [stackblitz/bolt.new#2834]
### Describe the bug

The website was working fine after a lot of prompts.

At some point I instructed Bolt to connect the actions in the website with the database so when I register a user is actually created in Supabase. Same with any other action in the website.

The changes were done but when the website is run it keeps loading forever within the website. It's  a spinner. It seems like the project is run but the website keeps on loading and it is never displayed.

There are no errors anywhere. UI nor console.

I reverted back and the website started working again. I reworded the prompt but the issues was created again.

I consumed al my daily credits trying to fix the issue, but no results. 

How can I continue with my project?

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-en91qx

### Steps to reproduce

1. Create a website with access to Supabase
2. Create a button to create any items (documents for example)
3. Add prompt to connect UI actions to the database

### Expected behavior

The expected behaivour is for the website to load correctly and at the end to save the data into the database.

### Screen Recording / Screenshot

![image](https://github.com/user-attachments/assets/2d50bf46-462b-4c84-b5cf-6e8a54dcf2ea)
![image](https://github.com/user-attachments/assets/dcdaef6f-4739-46c4-a0d2-2815905c3b8c)

### Platform

```
Browser name = Chrome
Full version = 131.0.0.0
Major version = 131
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 86426411,
  "usedJSHeapSize": 81188647,
  "jsHeapSizeLimit": 2160066560
}
Username = vanraidex
Chat ID = dd28a1b93514
```

**Comment by VanRaidex** [comment]
Just to keep updating this. It seems that is a problem with Chrome. I disabled all addons and still same issue.

I switched to Edge and all is working fine.

So there is definetly an issue with Chrome.

**Comment by VanRaidex** [comment]
It seems that it is other issue. Today, not even in Edge is working. It keeps loading forever again.

**Comment by VanRaidex** [comment]
Ok, found the real issue this time. I debugged the code and it was an issue with the Auth state. The code was not returning a value to the main call in App file so the spinner was loading forever. 

To fix it I told bolt that there was an issue with that and that no error was in console. I told him to add logs for us to see where the issue is.

After it did the fix I told him several times the issue was still persisting and each time I copied the logs printed. All of them.

After a few iterations doing the same, bolt found the issue and fix the auth code.
